lang/php7/Makefile

```diff
@@ -500,6 +500,12 @@ define Package/php7-fpm/install
 	$(INSTALL_BIN) ./files/php7-fpm.init $(1)/etc/init.d/php7-fpm
 endef
 
+define Package/php7-fpm/conffiles
+/etc/php7-fpm.conf
+/etc/php7-fpm.d/www.conf
```

Comment:
Please change to `/etc/php7-fpm.d/` instead.  If the user deposits any files in that directory, he'd likely expect __all or nothing__ propagation across a sysupgrade.  Doing some files but not others violates _The Principle of Least Astonishment_.

Reply:
@pprindeville : No problem. I am going to amend the last commit, so this message will probably be lost but at least you will get an email notification.